conf/log4j.properties.template

```diff
@@ -42,7 +42,7 @@ log4j.logger.org.apache.hadoop.hive.metastore.RetryingHMSHandler=FATAL
 log4j.logger.org.apache.hadoop.hive.ql.exec.FunctionRegistry=ERROR
 
 # For deploying Spark ThriftServer
-# SPARK-34128：Suppress undesirable TTransportException warnings involved in THRIFT-4805
+# SPARK-34128: Suppress undesirable TTransportException warnings involved in THRIFT-4805
```

Comment:
What's the typo, change here? I can't figure out what the diff is
But, we're about to overwrite this anyway for a log4j 2 upgrade

Reply:
The original `：` is a full-width character, but it should a half-width characters `:`. Feel free to close this minor fix if it's not necessary.